rest/resource-server/src/main/java/org/eclipse/sw360/rest/resourceserver/project/ProjectController.java

```diff
@@ -2061,6 +2066,57 @@ public void getlicenseClearingCount(HttpServletResponse response ,
         }
     }
 
+    @Operation(
+            description = "Get license obligations data from license database.",
+            tags = {"Project"}
+    )
+    @RequestMapping(value = PROJECTS_URL + "/{id}/licenseDbObligations", method = RequestMethod.GET)
+	public ResponseEntity<?> getLicObligations(@RequestParam(defaultValue = "0") int page,
```

Comment:
Can we use **Pageable** instead of **page** and **size**?

Reply:
Tried with Pageable but finding some challenges, therefore switched to page and size for time being.